configs/solo/solo_r50_fpn_3x_coco.py

```diff
@@ -0,0 +1,28 @@
+_base_ = './solo_r50_fpn_1x_coco.py'
+
+img_norm_cfg = dict(
+    mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+train_pipeline = [
+    dict(type='LoadImageFromFile'),
+    dict(type='LoadAnnotations', with_bbox=True, with_mask=True),
+    dict(
+        type='Resize',
+        img_scale=[(1333, 800), (1333, 768), (1333, 736), (1333, 704),
```

Comment:
considering using the 3x schedule in configs/common/

Reply:
follow the original implementation, will rerun and check this later